modules/auxiliary/scanner/misc/dahua_dvr_auth_bypass.rb

```diff
@@ -8,13 +8,14 @@ def initialize
       'Name'            => %q(Dahua DVR Auth Bypass Scanner),
       'Description'     => %q(Scans for Dahua-based DVRs and then grabs settings. Optionally resets a user's password and clears the device logs),
       'Author'          => [
+        'Tyler Bennett - Talos Consulting', # Metasploit module
         'Jake Reynolds - Depth Security', # Vulnerability Discoverer
-        'Tyler Bennett - Talos Infosec', # Metasploit Module
         'Jon Hart <jon_hart[at]rapid7.com>', # improved metasploit module
         'Nathan McBride' # regex extraordinaire
       ],
       'References'      => [
         [ 'CVE', '2013-6117' ],
+        [ 'URL', 'https://talosconsulting.net' ],
```

Comment:
Given that the references are intended to point people directly at an article or advisory, I think leaving this out until a full URL is available makes more sense.